src/main/kotlin/com/neva/gradle/fork/ForkExtension.kt

```diff
@@ -56,7 +56,7 @@ open class ForkExtension(val project: Project, val props: PropsExtension) {
     Properties().apply {
       load(file.bufferedReader())
     }.forEach { name, value ->
-      project.extensions.extraProperties.set(name.toString(), value)
+      project.extensions.extraProperties.set(name.toString(), props.encryptor.decrypt(value as String?))
```

Comment:
you are assuming auto-decryption for gradle.user.properties only?
it's good :D

Reply:
generally speaking ... current approach reflects approach used in gradle-credentials-plugin,
i didn't  want to invent sth different but rather stand on sth tested in battle;

but... maybe it's better ;)

Reply:
could you create corresponding PR for gradle-aem-multi which will use it?